doc/api/child_process.md

```diff
@@ -1094,6 +1095,27 @@ console.log(`Spawned child pid: ${grep.pid}`);
 grep.stdin.end();
 ```
 
+### subprocess.ref()
+<!-- YAML
+added: v0.7.10
+-->
+
+Calling `subprocess.ref()` after making a call to `subprocess.unref()` will
+restore the removed reference count for the child process, allowing the parent
```

Comment:
nit: `forcing` instead of `allowing` seems to be the right word here.

Reply:
OK